src/java/fr/paris/lutece/portal/business/user/authentication/LuteceDefaultAdminAuthentication.java

```diff
@@ -92,18 +98,18 @@ public String getAuthType( HttpServletRequest request )
     public AdminUser login( String strAccessCode, String strUserPassword, HttpServletRequest request )
         throws LoginException
     {
-        // Creating a record of connections log
-        UserLog userLog = new UserLog(  );
-        userLog.setAccessCode( strAccessCode );
-        userLog.setIpAddress( SecurityUtil.getRealIp( request ) );
-        userLog.setDateLogin( new java.sql.Timestamp( new java.util.Date(  ).getTime(  ) ) );
-
         // Test the number of errors during an interval of minutes
         int nMaxFailed = AdminUserService.getIntegerSecurityParameter( PROPERTY_MAX_ACCESS_FAILED );
         int nIntervalMinutes = AdminUserService.getIntegerSecurityParameter( PROPERTY_INTERVAL_MINUTES );
 
         if ( ( nMaxFailed > 0 ) && ( nIntervalMinutes > 0 ) )
         {
+            // Creating a record of connections log
+            UserLog userLog = new UserLog(  );
+            userLog.setAccessCode( strAccessCode );
+            userLog.setIpAddress( SecurityUtil.getRealIp( request ) );
+            userLog.setDateLogin( new java.sql.Timestamp( new java.util.Date(  ).getTime(  ) ) );
+
```

Comment:
Because the object is only used inside the if block, so no need to create in every cases